vue-handler.js

```diff
@@ -2,20 +2,20 @@ const Vue = require('vue');
 const vsr = require('vue-server-renderer');
 
 const typeToInfo = {
-    'pdf': ['fad fa-file-pdf', 'PDF', 'PDF file එක භාගත කරගන්න', 'application/pdf', 'darkred'],
-    'htm': ['fad fa-file-code', 'WEB', 'HTML file එක වෙත පිවිසෙන්න', 'text/html', 'blue'],
-    //'lin': ['fad fa-link', 'WWW', 'Link එක වෙත පිවිසෙන්න', ''], // redirect to url
-    'col': ['fad fa-folder', 'ගොනුව', 'ගොනුව වෙත පිවිසෙන්න', '', 'goldenrod'],
-    'zip': ['fad fa-file-archive', 'ZIP', 'ZIP file එක භාගත කරගන්න', 'application/zip', 'darkslategrey'],
-    'apk': ['fad fa-mobile-android', 'APP', 'android මෘදුකාංගය ලබාගන්න', 'application/octet-stream', 'green'],
-    'doc': ['fad fa-file-word', 'DOC', 'Word file එක භාගත කරගන්න', 'application/octet-stream', 'darkblue'],
-    'xls': ['fad fa-file-excel', 'EXCEL', 'Excel වගුව භාගත කරගන්න', 'application/octet-stream', 'darkgreen'],
-    'jpg': ['fad fa-file-image', 'IMAGE', 'පින්තූරය භාගත කරගන්න', 'image/jpeg', 'purple'],
-    'png': ['fad fa-file-image', 'IMAGE', 'පින්තූරය භාගත කරගන්න', 'image/png', 'purple'],
-    'txt': ['fad fa-file-alt', 'TXT', 'TEXT file එක වෙත පිවිසෙන්න', 'text/plain', 'black'],
-    'mp3': ['fad fa-file-audio', 'MP3', 'ධර්ම දේශනාව භාගත කරගන්න', 'audio/mpeg', 'navy'],
-    'm4a': ['fad fa-file-audio', 'M4A', 'ධර්ම දේශනාව භාගත කරගන්න', 'audio/mpeg', 'black'],
-    'unk': ['fad fa-file-alt', 'FILE', 'භාගත කරගන්න', 'application/octet-stream', 'darkgreen'], // unknown types
+    'pdf': ['fad fa-file-pdf', 'PDF', 'පීඩීඑෆ් ගොනුව බාගන්න', 'application/pdf', 'darkred'],
+    'htm': ['fad fa-file-code', 'WEB', 'HTML ගොනුව බලන්න', 'text/html', 'blue'],
+    //'lin': ['fad fa-link', 'WWW', 'සබැඳිය වෙත පිවිසෙන්න', ''], // redirect to url
+    'col': ['fad fa-folder', 'ගොනුව', 'බහාලුම අරින්න', '', 'goldenrod'],
+    'zip': ['fad fa-file-archive', 'ZIP', 'ZIP ගොනුව බාගන්න', 'application/zip', 'darkslategrey'],
+    'apk': ['fad fa-mobile-android', 'APP', 'ඇන්ඩ්‍රොයිඩ් යෙදුම ගන්න', 'application/octet-stream', 'green'],
+    'doc': ['fad fa-file-word', 'DOC', 'වර්ඩ් ගොනුව බාගන්න', 'application/octet-stream', 'darkblue'],
+    'xls': ['fad fa-file-excel', 'EXCEL', 'එක්සෙල් වගුව බාගන්න', 'application/octet-stream', 'darkgreen'],
+    'jpg': ['fad fa-file-image', 'IMAGE', 'රූපය බාගන්න', 'image/jpeg', 'purple'],
+    'png': ['fad fa-file-image', 'IMAGE', 'රූපය බාගන්න', 'image/png', 'purple'],
+    'txt': ['fad fa-file-alt', 'TXT', 'පෙළ ගොනුව බලන්න', 'text/plain', 'black'],

```

Comment:
Word, Excel, PDF are best to keep in English

Reply:
Agree. 
And පෙළ has other meanings. So it is better to keep as TEXT file.
බාගත කරගන්න/භාගත කරගන්න sounds more formal on the website.